lightautoml/text/abbreviations.py

```diff
@@ -0,0 +1,173 @@
+ABBREVIATIONS = {
```

Comment:
Do we need that dict in the library? I would like to put it into custom config file.